mismi-ec2/test/Test/IO/Mismi/EC2/Arbitrary.hs

```diff
@@ -9,10 +9,6 @@ import           P
 
 import           Test.QuickCheck
 
-instance Arbitrary VirtualizationType where
-  arbitrary =
-    elements [HVM, Paravirtual]
-
 instance Arbitrary SecurityGroupName where
```

Comment:
Shouldn't this go to core arbitrary as well (and possibly others). Plus can this pull in the core instances so you dont have to depend on both everytime.